frontend/src/components/Modal/ResourceModal/index.jsx

```diff
@@ -25,13 +25,19 @@ const ResourceModal = props => {
   };
 
   const handleEditResource = async data => {
-    data.tags = data.tags.split(",").map(tag => tag.trim());
+    data.tags = data.tags
+      .split(",")
+      .filter(t => t)
+      .map(tag => tag.trim());
     await editAndRefreshResource(data, props.resource._id);
     props.closeModal();
   };
 
   const handleAddResource = async data => {
-    data.tags = data.tags.split(",").map(tag => tag.trim());
+    data.tags = data.tags
+      .split(",")
+      .filter(t => t)
```

Comment:
is this `.filter(t => t)` necessary?

Reply:
Good question but yes - that serves the purpose of filtering out the falsy, empty-string values which will cause an issue if we try to send, say `[""]` to our backend instead of just `[]` for the tag array.

Reply:
sounds good - can we get that in a comment on our code? I feel like someone might see this in a future iteration and get confused about its usage